modules/form-field/css/lib/form-field.scss

```diff
@@ -70,12 +69,17 @@
 }
 
 .wdc-form-group {
+  padding: 0;
+  margin: 0;
+  border: 0;
+
+  margin-bottom: $wdc-spacing-m;
+
   @include wdc-form-group();
 }
 
 @mixin wdc-form-label-position-left() {
   .wdc-form-field-wrapper {
-    display: flex;
```

Comment:
Why did we drop `display: flex` and position the label absolutely? Can't this be laid out as flexbox the same as React?

Reply:
React:
```
<fieldset>
  <legend>
  <wrapper>
    <radiogroup>
    <hint>
  <wrapper>
</fieldset>
```

CSS:
```
<fieldset>
  <legend>
  <radiogroup>
  <hint>
</fieldset>
```

In React, we have a wrapping around `RadioGroup` and `Hint`, but not in CSS, so I had to refactor Label to allow us to have it be positioned properly for `RadioGroup`, `Checkbox` and `TextInput` (in CSS).

The alternative would have been to refactor the HTML to be closer to React, but it would mean having an extra div.

If I leave `display:flex;`, the error/alert border of the `RadioGroup` will grow.

Without `display: flex;`: 
![image](https://user-images.githubusercontent.com/723109/66229824-b5308800-e6b0-11e9-9819-623e0dfbf5ca.png)

With `display: flex;`:
![image](https://user-images.githubusercontent.com/723109/66229843-c083b380-e6b0-11e9-84c0-b51ba19c3211.png)

Reply:
Ah, I see what you mean. I missed the fact that the HTML structure was slightly different.

Hmm 🤔. I get that this works, it just feels rather brittle to absolutely position it. It's also a shame that we're doing this for all labels/fields even though it's only an issue for `.wdc-form-group-fields` (for other input types, the hint message is inside `.wdc-form-field`).

I feel like in the future we're going to need to support variable width labels, and flexbox just feels like a more "flexible" option (ha). How do you feel about adding an extra wrapper around the group and hint in CSS so that it matches React so that the styling can be more consistent with the other input types when there's an alert/error?

Reply:
The issue is that if I switch back to the way it was before, with `float: left;` on the Label, then it doesn't know the height of its parent component and I can't center it vertically.

Reply:
This change also allows us to properly position `label` for form field that have a height of `24px` or `40px or higher`. If we change it back to what it was before for other fields than `.wdc-form-group-fields`, we would lose this ability.

We should revisit this in the future when we'll support variable width labels.